### DISCUSSION TITLE
Развернуть дубль clickhouse из бэкапа

### DISCUSSION THREAD

**Discussion by SergeyDimov** [Altinity/clickhouse-backup#461] [Q&A] [answered]
Всем привет!
У меня есть clickhouse, который развернут в k3s (при этом папка /var/lib/clickhouse/ прокинута в виде вольюма на локальную машину)
Я делаю бэкап сервиса при помощи утилиты clickhouse-backup (запускаю утиллиту на другой машине, указав адрес сервиса), при этом создаются директории мета (папка shadow создаётся только в прокинутом вольюме). При попытке развернуть копию сервиса из бэкапа в другом контейнере. Восстанавливаются таблицы, но они пустые...
Вопрос можно ли как то на основании бэкапа развернуть копию сервиса в другом контейнере?

Hi all!
I have a clickhouse that is deployed in k3s (with the /var/lib/clickhouse/ folder thrown in the form of a volume on the local machine)
I backup the service using the clickhouse-backup utility (I run the utility on another machine, specifying the address of the service), and meta directories are created (the shadow folder is created only in the forwarded volume). When trying to deploy a copy of a service from a backup in another container. Tables are restored, but they are empty...
The question is whether it is possible to deploy a copy of the service in another container based on the backup?

Clickhouse ver. 22.1.3.7
Clickhouse-backup ver. 1.2.0

**Comment by Slach** [comment, marked as answer]
> запускаю утиллиту на другой машине, указав адрес сервиса

так работать не будет
clickhouse-backup необходим доступ до /var/lib/clickhouse потому что из него берутся данные
запускайте clickhouse-backup как отдельный контейнер в том же Pod что и clickhouse-server

**Reply by SergeyDimov** [reply]
Понял, спасибо.
